index.html

```diff
@@ -5,8 +5,39 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Sam - Portofolio</title>
+    <script src="https://kit.fontawesome.com/b39c945343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="assets/style.css" />
 </head>
 <body>
-    
+    <header id="header_menu">
+        <a class="welcome_text">
+            Welcome
+        </a>
+        <nav class="toggleNav">
+            <img src="assets/icons/union.png" alt="union icon" />
+        </nav>
+    </header>
+    <section id="main_section">
+        <h1 id="presentation_text">
+            <span class="greetings">Hey there. I’m Sammy</span> <br>
+            <span class="profession">I’m a software developer</span></h1>
+        <p id="biography">
+             I can help you build a dream product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to reach out to me.
+        </p>
+        <ul id="social_media_links">
```

Comment:
- [ ] [Optional] Consider using the social icons from the Figma template.

## Current:
![image](https://user-images.githubusercontent.com/92215878/191732171-57bd359a-6b43-49d6-ba4f-0017bd8fb6ed.png)

##  Figma:
![image](https://user-images.githubusercontent.com/92215878/191732318-fa62565a-18b0-44f8-8e94-e3dbb1d98a66.png)